lib/elixir/lib/string.ex

```diff
@@ -241,7 +241,8 @@ defmodule String do
   @doc """
   Divides a string into substrings at each Unicode whitespace
   occurrence with leading and trailing whitespace ignored. Groups
-  of whitespace are treated as a single occurrence.
+  of whitespace are treated as a single occurrence. Divisions do
+  not occur on whitespace marked as no-break by unicode.
```

Comment:
Maybe we can say "on whitespace marked as no-break by unicode" –> "non-breakable whitespace"?